examples/petstore.oas2.yaml

```diff
@@ -1,700 +1,19 @@
 swagger: '2.0'
-info:
```

Comment:
@XVincentX, why are we removing this?

Reply:
Argh, I must have done something wrong here. I'll restore it.